pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.kryptokrauts</groupId>
   <artifactId>contraect-maven-plugin</artifactId>
-  <version>1.0.0</version>
+  <version>1.0.1-SNAPSHOT</version>
```

Comment:
for the release build we need `1.0.1` here

Reply:
updated version to 1.1.0
closes #39 
closes #40